src/explore-education-statistics-common/src/components/ScreenReaderMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+
+/**
+ * Announces changes to screen readers.
+ * Use when just using aria-live on an existing element won't work
+ * because it announces changes multiple times.
+ */
+export default function ScreenReaderMessage({ message }: { message: string }) {
+  const [screenReaderMessage, setScreenReaderMessage] = useState('');
+
+  useEffect(() => {
+    // Clear the message then repopulate to ensure the new message is read,
+    // and only read once.
+    setScreenReaderMessage('');
+    setTimeout(() => {
+      setScreenReaderMessage(message);
+    }, 200);
```

Comment:
Might want to write a little test suite to check that this stuff happens?

Reply:
Good point, I've added tests now.